Can't compile

fatal: Не найден git репозитоий (или один из его каталогов вплоть до точки монтирования /var/tmp/portage)
Останавливаю поиск на границе файловой системы (так как GIT_DISCOVERY_ACROSS_FILESYSTEM не установлен).
grep: ./debian/changelog: Нет такого файла или каталога
#test -e ./altyo && rm ./altyo
/usr/bin/valac-0.36 -o altyo  -v --disable-warnings -X -DGETTEXT_PACKAGE=\"altyo\" -X -DAY_GIT_HASH=\"\" -X -DAY_CHANGELOG_TAG=\"\" --vapidir ./vapi --pkg gtk+-3.0 --pkg gdk-x11-3.0 --pkg cairo --pkg posix --pkg gmodule-2.0 -D VTE_2_91 --pkg vte-2.91 vapi/config.vapi vapi/xkb.vapi main.vala hvbox.vala altyo_terminal.vala altyo_window.vala altyo_hotkey.vala altyo_config.vala altyo_settings.vala data/altyo.c
Loaded package `/usr/share/vala-0.36/vapi/glib-2.0.vapi'
Loaded package `/usr/share/vala-0.36/vapi/gobject-2.0.vapi'
Loaded package `/usr/share/vala-0.36/vapi/gtk+-3.0.vapi'
Loaded package `/usr/share/vala-0.36/vapi/gio-2.0.vapi'
Loaded package `/usr/share/vala-0.36/vapi/atk.vapi'
Loaded package `/usr/share/vala-0.36/vapi/cairo.vapi'
Loaded package `/usr/share/vala-0.36/vapi/gdk-pixbuf-2.0.vapi'
Loaded package `/usr/share/vala-0.36/vapi/gdk-3.0.vapi'
Loaded package `/usr/share/vala-0.36/vapi/pango.vapi'
Loaded package `/usr/share/vala-0.36/vapi/pangocairo.vapi'
Loaded package `/usr/share/vala-0.36/vapi/x11.vapi'
Loaded package `/usr/share/vala-0.36/vapi/gdk-x11-3.0.vapi'
Loaded package `/usr/share/vala-0.36/vapi/posix.vapi'
Loaded package `/usr/share/vala-0.36/vapi/gmodule-2.0.vapi'
Loaded package `/usr/share/vala/vapi/vte-2.91.vapi'
altyo_window.vala:2894.99-2894.119: error: Argument 1: Cannot convert from `uint' to `int'
        string s = ( tab.tbutton.tab_title != null ? tab.tbutton.tab_title : _("index %d").printf(tab.tbutton.tab_index) );
                                                                                                  ^^^^^^^^^^^^^^^^^^^^^
altyo_settings.vala:1636.107-1636.110: error: Argument 1: Cannot convert from `uint' to `int'
                string msg=_("New style will not be saved!\nin line %d  at position %d\nerror:%s").printf(line,pos,S);
                                                                                                          ^^^^
altyo_settings.vala:713.24-713.27: error: Argument 1: Cannot pass value to reference or output parameter
          store.remove(iter);
                       ^^^^
altyo_settings.vala:747.24-747.27: error: Argument 1: Cannot pass value to reference or output parameter
          store.remove(iter);
                       ^^^^
altyo_settings.vala:781.24-781.27: error: Argument 1: Cannot pass value to reference or output parameter
          store.remove(iter);
                       ^^^^
altyo_settings.vala:968.59-968.62: error: Argument 1: Cannot convert from `uint' to `int'
      msg=_("in line %d  at position %d error:%s").printf(line,pos,S);
                                                          ^^^^
Compilation failed: 6 error(s), 0 warning(s)
make: *** [Makefile:79: default] Ошибка 1

--- a/altyo_settings.vala
+++ b/altyo_settings.vala
@@ -710,7 +710,7 @@ public class AYSettings : AYTab{
         view.get_cursor(out path,out s_column);
         if(store.get_iter(out iter,path))
         if(!store.iter_has_child(iter)){
-          store.remove(iter);
+          store.remove(ref iter);
           if(store.get_iter(out iter,path))
             view.set_cursor(path,null,false);
           else if(path.prev())
@@ -744,7 +744,7 @@ public class AYSettings : AYTab{
         view.get_cursor(out path,out s_column);
         if(store.get_iter(out iter,path))
         if(!store.iter_has_child(iter)){
-          store.remove(iter);
+          store.remove(ref iter);
           if(store.get_iter(out iter,path))
             view.set_cursor(path,null,false);
           else if(path.prev())
@@ -778,7 +778,7 @@ public class AYSettings : AYTab{
         view.get_cursor(out path,out s_column);
         if(store.get_iter(out iter,path))
         if(!store.iter_has_child(iter)){
-          store.remove(iter);
+          store.remove(ref iter);
           if(store.get_iter(out iter,path))
             view.set_cursor(path,null,false);
           else if(path.prev())
@@ -965,7 +965,7 @@ public class AYSettings : AYTab{
     string msg="";
     uint line,pos;
     if(!this.check_css(B.buffer.text,ref S,out line,out pos)){
-      msg=_("in line %d  at position %d error:%s").printf(line,pos,S);
+      msg=_("in line %d  at position %d error:%s").printf((int)line,(int)pos,S);
       debug("on_check_css_button_activate %s",msg);
       TextIter where;
       B.buffer.get_iter_at_line_offset(out where,(int)line,(int)pos);
@@ -1633,7 +1633,7 @@ public class AYSettings : AYTab{
               string S="";
               uint line,pos;
               if(!this.check_css(s,ref S,out line,out pos)){
-                string msg=_("New style will not be saved!\nin line %d  at position %d\nerror:%s").printf(line,pos,S);
+                string msg=_("New style will not be saved!\nin line %d  at position %d\nerror:%s").printf((int)line,(int)pos,S);
                 debug("on config apply css error %s",msg);
                 this.ayobject.main_window.show_message_box(_("AltYo CSS style error"),msg);
               }else{//looks good

--- a/altyo_window.vala
+++ b/altyo_window.vala
@@ -2891,7 +2891,7 @@ public class AYObject :Object{
       for (item_it = this.children_removed.last(); item_it != null; item_it = item_it.prev) {
         AYTab tab = item_it.data;
         tab.stop_remove_timer();//disable timer while popup shown
-        string s = ( tab.tbutton.tab_title != null ? tab.tbutton.tab_title : _("index %d").printf(tab.tbutton.tab_index) );
+        string s = ( tab.tbutton.tab_title != null ? tab.tbutton.tab_title : _("index %d").printf((int)tab.tbutton.tab_index) );
         menuitem = new Gtk.MenuItem.with_label ("%d: %s".printf(index,s));
         menuitem.activate.connect(()=>{
           debug("trying to restore tab %s",menuitem.label);

